P0009/P0009.bs

```diff
@@ -960,35 +952,32 @@ constexpr extents& operator=(const extents<OtherExtents...>& other);
 static constexpr size_t rank() const noexcept;
 ```
 
-* *Returns:* `sizeof...(Extents)`
+* *Returns:* `sizeof...(StaticExtents)`
 
 <br/>
 ```c++
 static constexpr size_t rank_dynamic() const noexcept;
 ```
 * *Returns:* 
-    * If `sizeof...(Extents)==0` is `true`, then zero;
-    * Otherwise, `((Extents==dynamic_extent)+...)`. 
-
-*[Note:* This is the number of dynamic extents. *—end note]*
+    * If `sizeof...(StaticExtents)==0` is `true`, then zero;
+    * Otherwise, `((StaticExtents==dynamic_extent)+...)`. 
 
 <br/>
 ```c++
 static constexpr index_type static_extent(size_t r) const noexcept;
 ```
 
 * *Returns:* 
-    * If 0 <= `r` < `rank()`, then the `r`th entry 
-      in the `Extents` parameter pack;
+    * If `r` is in the range `[0,rank())`, then the r-th entry 
```

Comment:
It's OK to put ranges in code font.  See http://eel.is/c++draft/partial.sum#2 .  However, there should be a space after the comma, here and elsewhere.